src/widgets/settingspages/CommandPage.hpp

```diff
@@ -11,9 +12,11 @@ class CommandPage : public SettingsPage
 {
 public:
     CommandPage();
+    bool handleCommandDuplicates(EditableModelView *view);
```

Comment:
If we need to act on the model view, store it as a private variable instead so you can access it with `this->view`.
Then instead of a public `handle..` function, make it a private `checkCommandDuplicates` function that makes all changes necessary